examples/quickstarts/cakephp-mysql-persistent.json

```diff
@@ -17,8 +17,8 @@
   },
   "message": "The following service(s) have been created in your project: ${NAME}, ${DATABASE_SERVICE_NAME}.\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/openshift/cake-ex/blob/master/README.md.",
   "labels": {
-    "template": "cakephp-mysql-persistent",
-    "app": "cakephp-mysql-persistent"
+      "template": "cakephp-mysql-persistent",
```

Comment:
these whitespace changes don't appear to be visible upstream?

Reply:
also came from SCL, I may have missed that they didn't run the import script on the PR but it'll get resolved next time around I suppose.